meta-client/src/main/java/org/apache/helix/metaclient/constants/MetaClientConstants.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.helix.metaclient.constants;
+
+public interface MetaClientConstants {
+
+  // Stop retrying when we reach timeout
+  int DEFAULT_OPERATION_RETRY_TIMEOUT = Integer.MAX_VALUE;
```

Comment:
I think this value is inherited. But could we use a smaller timeout value for this? Open to discussion on this.

Reply:
If we change to a smaller number, will it change user behavior when user witch to metaclient?

Reply:
Chatted offline, there is a similar change in current behavior https://github.com/apache/helix/pull/2293 
A shorter default timeout makes more sense and we should keep in sync